src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Components/ComponentDesignTimeNodeWriter.cs

```diff
@@ -611,6 +611,10 @@ private void WriteTypeInferenceMethodParameterInnards(CodeRenderingContext conte
                 // first, and that's where it's populated. However if this goes wrong somehow, we don't want to
                 // throw, so use a fallback
                 context.CodeWriter.Write(syntheticArg.ValueExpression ?? "default");
```

Comment:
It seems like we should capture what was actually written; let's make sure we have a test that hits the `default` part of this example.